back/engines/commercial/content_builder/config/initializers/pg_search.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# https://github.com/Casecommons/pg_search/issues/252#issuecomment-486606367
+module PgSearchPatch
+  def full_name
+    if @column_name.include?('"') || @column_name.include?("'")
```

Comment:
Have you seen [this comment](https://github.com/Casecommons/pg_search/issues/252#issuecomment-840959889) in the GH issue? It looks a bit cleaner to me then trying to parse quotes. What do you think?

Reply:
I agree. Done.